tests/Unit/Core/Variables/StandardVariableProviderTest.php

```diff
@@ -7,6 +7,9 @@
  */
 
 use TYPO3Fluid\Fluid\Core\Variables\StandardVariableProvider;
+use TYPO3Fluid\Fluid\Core\Variables\VariableExtractor;
```

Comment:
This class is unused, please remove this.

Reply:
I've removed the use statement